scripts/doc_crosslinks/generate_cross_link_data.py

```diff
@@ -25,10 +28,43 @@ def generateDocsMap(apiDefinitionsPath, apiDefinitionsRelativeFilePath):
             apiContent = json.loads(apiDefinition.read())
             if "uid" in apiContent["metadata"].keys():
                 sdks[apiContent["metadata"]["uid"]] = getServiceName
-            clientClass[apiContent["metadata"]["uid"]] = getClientClassNameFromMetadata(apiContent["metadata"])
-
+            clientClassPrefix[apiContent["metadata"]["uid"]] = getClientClassNameFromMetadata(apiContent["metadata"])
+            asyncOnlyOps = {}
+            for op in apiContent['operations'].keys():
```

Comment:
Can we move this to separate function something like updateAsyncOperations ?

Reply:
Done